packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -338,7 +338,7 @@ export class Cluster extends Resource implements ICluster {
     const autoScalingGroup = new autoscaling.AutoScalingGroup(this, id, {
       vpc: this.vpc,
       machineImage,
-      updateType: options.updateType || autoscaling.UpdateType.REPLACING_UPDATE,
+      updateType: !!options.updatePolicy ? undefined : options.updateType || autoscaling.UpdateType.REPLACING_UPDATE,
```

Comment:
This behavior change should also be reflected in the `@default` docstring of the `updateType` field.

Reply:
Is this really changing the default for the field, or am I just changing what gets passed in?

Reply:
```
`updatePolicy` takes precedence over `updateType`.

@default UpdateType.REPLACING_UPDATE, unless updatePolicy has been set
```

Reply:
Thanks. I've updated this PR to reflect the requested changes.